client/src/pages/_app.tsx

```diff
@@ -0,0 +1,21 @@
+import React, { ReactElement } from 'react';
+
+import type { AppProps } from 'next/app';
+import Head from 'next/head';
+
+const App = (props: AppProps): ReactElement => {
+  const { Component, pageProps } = props;
+
+  return (
+    <>
+      <Head>
+        <meta charSet="utf-8" />
+        <link rel="shortcut icon" href="https://developersam.com/favicon.ico" />
```

Comment:
Copied from https://github.com/SamChou19815/website/blob/master/packages/dot-dev/pages/_app.tsx 🤣 

Probably will keep it until we made a favicon.